pegasus/migrations/093_add_hashed_email_to_contacts.rb

```diff
@@ -0,0 +1,30 @@
+require 'digest/md5'
+
+Sequel.migration do
+  up do
+    add_column :contacts, :hashed_email, String, null: false
+
+    batch_update do |contact|
+      self[:contacts].where(id: contact[:id]).update(hashed_email: Digest::MD5.hexdigest(contact[:email].downcase))
+    end
+  end
+
+  down do
+    drop_column :contacts, :hashed_email
+  end
+end
+
+# Update rows in batches.
+# (adapted from pegasus/migrations/077_add_abuse_score_column_to_storage_apps.rb)
+def batch_update
```

Comment:
Not that important, but if we're using this a lot (as it seems we are), it might be worth extracting into a module that can be included.